src/sssom/context.py

```diff
@@ -108,8 +122,12 @@ def get_default_metadata() -> Metadata:
             if "@id" in v and "@prefix" in v:
                 if v["@prefix"]:
                     prefix_map[key] = v["@id"]
+    del prefix_map["@vocab"]
 
     prefix_map.update({(k, v) for k, v in contxt_external.items() if k not in prefix_map})
+    clean_dc(prefix_map)
+    # Tests if the prefix map is a valid bijective map
+    Converter.from_prefix_map(prefix_map)
```

Comment:
How does this line of code affect the rest of the code? It seems a bit disconnected?

Reply:
it will throw an error if the prefix map is not bijective, I can update the comment to make this more clear

Reply:
Ah cool. No this makes sense now!

Reply:
I added a more explicit helper function for checking prefix map validity in 439f561